test/integration/cms/api/sections_test.rb

```diff
@@ -60,6 +60,17 @@ def create_section
         )
       end
 
+      test 'index filters by parent_id' do
+        parent = FactoryBot.create(:cms_section, provider: @provider, parent: @provider.sections.root)
+        5.times { FactoryBot.create(:cms_section, provider: @provider, parent: @provider.sections.root) }
+        2.times { FactoryBot.create(:cms_section, provider: @provider, parent: parent) }
```

Comment:
`create_list` suggestion again:

```suggestion
        FactoryBot.create(:cms_section, 5, provider: @provider, parent: @provider.sections.root)
        FactoryBot.create(:cms_section, 2, provider: @provider, parent: parent)
```

Reply:
Done